scripts/check-includes.py

```diff
@@ -15,10 +15,10 @@
 def main(argv):
     argparser = ArgumentParser()
     argparser.add_argument('--generated-includes-dir', action='append',
-                           help='Directory where generated includes are located.')
-    argparser.add_argument('--file', type=open, help='File to check.')
+                           help='Directory where generated includes are located.',required=True)
+    argparser.add_argument('--file', type=open, help='File to check.',required=True)
```

Comment:
better make it a positional argument `add_argument('file')` since it is required anyway